packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -168,6 +177,10 @@ export type BaseInputProps = FormInputLabelProps &
      * Sets the textarea's number of lines
      */
     numberOfLines?: 2 | 3 | 4 | 5;
+    /**
+     * Sets the accessibility label for the input
+     */
+    accessibilityLabel?: string;
```

Comment:
Need to pass accessibility label for individual otp input fields as "Character 1", "Character 2", etc.